website_sale_product_detail_attribute_image/tests/test_website_sale_product_detail_attribute_image.py

```diff
@@ -4,14 +4,89 @@
 
 
 class WebsiteSaleHttpCase(HttpCase):
+    at_install = False
+    post_install=True
```

Comment:
It's better that you use the provided decorators instead, because this has changed in v12 and those decorators provide compatibility.